veggieseasons/lib/screens/details.dart

```diff
@@ -4,18 +4,156 @@
+      child: DecoratedBox(
+        decoration: BoxDecoration(
+          color: Styles.seasonColors[season],
+          borderRadius: BorderRadius.circular(25.0),
+          border: Styles.seasonBorder,
+        ),
+        child: SizedBox(
+          height: 50.0,
+          width: 50.0,
+          child: Center(child: Text(_firstChars, style: Styles.seasonText)),
+        ),
+      ),
+    );
+  }
+}
+
 class DetailsScreen extends StatelessWidget {
+  final int id;
+
+  DetailsScreen({@required this.id});
+
+  Widget _createFavoriteButton(bool isFav, VoidCallback onPressed) {
+    return CupertinoButton(
+      color: Styles.buttonColor,
+      child: Row(
+        mainAxisSize: MainAxisSize.min,
+        children: [
+          Icon(
+            isFav ? Styles.checkedIcon : Styles.uncheckedIcon,
+            color: Styles.buttonIconColor,
+          ),
+          SizedBox(width: 4.0),
+          Text(isFav ? 'Saved to Garden' : 'Save to Garden'),
+        ],
+      ),
+      onPressed: onPressed,
+    );
+  }
+
+  Widget _createHeader(AppState model) {
+    final veggie = model.getVeggie(id);
+
+    return SizedBox(
+      height: 200.0,
+      child: Stack(
+        children: [
+          Positioned(
+            right: 0.0,
+            left: 0.0,
+            child: Image.asset(
+              veggie.imageAssetPath,
+              fit: BoxFit.cover,
+            ),
+          ),
+          Positioned(
+            bottom: 0.0,
+            left: 0.0,
+            right: 0.0,
+            child: DecoratedBox(
+              decoration: BoxDecoration(
+                gradient: Styles.shadowGradient,
+              ),
+              child: Padding(
+                padding: const EdgeInsets.fromLTRB(16.0, 50.0, 16.0, 16.0),
+                child: Text(
+                  veggie.name,
+                  style: Styles.subheadText,
+                ),
+              ),
+            ),
+          ),
+        ],
+      ),
+    );
+  }
+
+  Widget _createDetails(AppState model) {
+    final veggie = model.getVeggie(id);
+
+    return Padding(
+      padding: const EdgeInsets.symmetric(
+        horizontal: 24.0,
+        vertical: 24.0,
+      ),
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.stretch,
+        children: <Widget>[
+          Row(
+            mainAxisSize: MainAxisSize.max,
+            children: [
+              Wrap(
+                children:
+                    veggie.seasons.map<Widget>((s) => SeasonCircle(s)).toList(),
+              ),
+              SizedBox(width: 8.0),
+              Expanded(
+                child: Align(
+                  alignment: Alignment.centerRight,
+                  child: Text(
+                      veggieCategoryNames[veggie.category].toUpperCase(),
+                      style: Styles.minorText),
+                ),
+              ),
+            ],
+          ),
+          SizedBox(height: 10.0),
+          Text(veggie.shortDescription),
```

Comment:
Instead of two SizedBoxes wrap Text with padding? 
Padding(padding: EdgeInsets.symmetric(vertical: 10.0), child: Text(veggie.shortDescription))

Reply:
I normally love my sized boxes, but a two-for-one is a good idea.